### PR TITLE
Bump pygmt from v0.2.0 to v0.2.1

### PR DESCRIPTION
**Pull request by weiji14**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -4,5 +4,5 @@ channels:
 dependencies:
   - python=3.8
```
Comment (weiji14) [inline environment.yml:5]:
```suggestion
  - python=3.9
```

File: environment.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: try-gmt
 channels:
   - conda-forge
 dependencies:
-  - python=3.8
+  - python=3.9
```
Comment (weiji14) [inline environment.yml:5]:
```suggestion
  - python=3.8
```

**Comment by weiji14** [comment]
@seisman, not sure what's up with the repo2docker Validator, but have a go at it in the morning when you're awake.

**Review by seisman — APPROVED** [review]

**Comment by seisman** [comment]
> @seisman, not sure what's up with the repo2docker Validator, but have a go at it in the morning when you're awake.

Perhaps because you're using your own fork?

**Pull request merged**